classes/Avif/Avif.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace Imagify\Avif;
+
+use Imagify\Traits\InstanceGetterTrait;
+
+/**
+ * AVIF image
+ */
+class Avif {
+	use InstanceGetterTrait;
+
+	/**
+	 * Class init: launch hooks.
+	 *
+	 * @since 2.2
+	 */
+	public function init() {
+		add_action( 'update_option_imagify_settings', [ $this, 'maybe_bulk_optimize_callback' ] );
```

Comment:
I'm thinking this could directly be in the Bulk class instead, it would make more sense

Reply:
Done